examples/hal/compute/main.rs

```diff
@@ -0,0 +1,162 @@
+extern crate env_logger;
+extern crate gfx_hal as hal;
+#[cfg(feature = "dx12")]
+extern crate gfx_backend_dx12 as back;
+#[cfg(feature = "vulkan")]
+extern crate gfx_backend_vulkan as back;
+#[cfg(feature = "metal")]
+extern crate gfx_backend_metal as back;
+#[cfg(feature = "gl")]
+extern crate gfx_backend_gl as back;
+#[cfg(feature = "gl")]
+extern crate glutin;
+#[cfg(feature = "gl")]
+use glutin::HeadlessRendererBuilder;
```

Comment:
this is for OSMesa runs, it's not going to support compute. You can safely just ignore GL for now.